packages/playground/blueprints/src/lib/steps/run-wp-installation-wizard.ts

```diff
@@ -0,0 +1,33 @@
+import { UniversalPHP } from '@php-wasm/universal';
+
+export interface WordPressInstallationOptions {
+	adminUsername?: string;
+	adminPassword?: string;
+}
+
+/**
+ * Installs WordPress
+ *
+ * @param playground The playground client.
+ * @param options Installation options.
+ */
+export async function runWpInstallationWizard(
+	playground: UniversalPHP,
+	options: WordPressInstallationOptions
+) {
+	await playground.request({
+		url: '/wp-admin/install.php?step=2',
+		method: 'POST',
+		formData: {
+			language: 'en',
+			prefix: 'wp_',
+			weblog_title: 'My WordPress Website',
+			user_name: options.adminPassword || 'admin',
+			admin_password: options.adminPassword || 'password',
+			admin_password2: options.adminPassword || 'password',
```

Comment:
what's `password2`?

Reply:
Installation wizard demands typing the same password twice

Reply:
ahhhhh. sneaky.

Reply:
Documented in 43d02caf